modules/twinklespeedy.js

```diff
@@ -248,6 +251,21 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 			}
 		]
 	});
+	tagOptions.append({
+		type: 'checkbox',
+		list: [
+			{
+				label: 'Tag for creation protection (salting) as well',
+				value: 'salting',
+				name: 'salting',
+				tooltip: 'When selected, the speedy deletion tag will be accompanied by a {{salt}} tag requesting that the deleting administrator apply creation protection. Only select if this page has been repeatedly recreated.',
+				disabled: isSysop && !Twinkle.getPref('deleteSysopDefaultToTag'),
+				event: function(event) {
+					event.stopPropagation();
+				}
+			}
+		]
+	});
```

Comment:
Put the three checkbox configs in one list. `tagOptions.append({ type: 'checkbox', list: [ { ... checkbox 1 ... }, { ... checkbox 2 ... } ]` rather than in 3 separate ones.

Also, I think the "Tag with multiple criteria" option should be at the bottom.

Reply:
>Also, I think the "Tag with multiple criteria" option should be at the bottom

OOS for this, but I think that'd be better discussed separately if it's a good idea; delete and tag are somewhat misaligned after #563.

Reply:
I don't know what "OOS" means. 

I meant the new checkbox should probably come above the the "Tag with multiple criteria" checkbox.

Reply:
Moved above

Reply:
Ah gotcha, that makes sense.